Makefile

```diff
@@ -1,3 +1,18 @@
+# Current Operator version
+VERSION_DATE ?= $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
+VERSION_PKG ?= "github.com/os-observability/tempo-operator/internal/version"
+TEMPO_VERSION ?= "1.5.0"
+OPERATOR_VERSION ?= 0.0.1
```

Comment:
Create an issue to replace this by `shell git describe --tags --always --dirty="-dev"` once we have the first tag. Otherwise, `make bundle` will fail always

Reply:
afaics it should also work in repos without tags, `--always` uses the last commit id as a fallback if the repo doesn't contain any tags. If I run it in my tempo-operator fork, it prints the last commit id.

Reply:
Yes but the problem is when generating the bundle. It only accepts versioning using `x.y.z`. The commit SHA is not following that structure and the bundle generation fails.

Reply:
Ah! Makes sense.
I didn't realize that there is a strict validation of the version number format.